content/applications/hr/referrals.rst

```diff
@@ -562,13 +407,13 @@ Create rewards
 The rewards are the only configurations needed when setting up the Referrals application. Only users
```

Comment:
```suggestion
The rewards are the only configurations needed when setting up the *Referrals* application. Only users
```
again, please go through this PR to ensure all application titles are formatted correctly. I will stop doing them personally from here on out --  but please double-check their accuracy during your revisions. thanks!